Content.Shared/DeltaV/Implants/AddFactions/AddFactionsImplantSystem.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Content.Shared.Implants;
+using Content.Shared.NPC.Systems;
+using Robust.Shared.Containers;
+
+namespace Content.Shared.DeltaV.Implants.AddFactions;
+
+public sealed class AddFactionsImplantSystem : EntitySystem
+{
+    [Dependency] private readonly NpcFactionSystem _npc = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<AddFactionsImplantComponent, ImplantImplantedEvent>(OnImplantImplantedEvent);
+        SubscribeLocalEvent<AddFactionsImplantComponent, EntGotRemovedFromContainerMessage>(OnRemove);
+    }
+
+    private void OnImplantImplantedEvent(Entity<AddFactionsImplantComponent> ent, ref ImplantImplantedEvent args)
+    {
+        if (args.Implanted == null)
```

Comment:
`if (args.Implanted is not {} target)`

then use target instead of args.Implanted.Value